ts/components/ContextualHelpModal.tsx

```diff
@@ -43,101 +45,110 @@ const styles = StyleSheet.create({
+      animationType={props.modalAnimation || "slide"}
+      transparent={true}
+      onDismiss={onClose}
+      onRequestClose={onClose}
+    >
+      <Container>
+        <BaseHeader
+          headerTitle={I18n.t("contextualHelp.title")}
+          customRightIcon={{ iconName: "io-close", onPress: onClose }}
+        />
 
-          {!this.state.content && (
-            <View centerJustified={true}>
-              <ActivityIndicator color={themeVariables.brandPrimaryLight} />
-            </View>
-          )}
-          {this.state.content && (
-            <Content
-              contentContainerStyle={styles.contentContainerStyle}
-              noPadded={true}
-            >
-              <H3>{this.props.title}</H3>
-              <View spacer={true} />
-              {this.state.content}
-              <View spacer={true} />
-              {this.props.faqCategories &&
-                this.props.contentLoaded && (
-                  <FAQComponent
-                    onLinkClicked={this.props.onLinkClicked}
-                    faqCategories={this.props.faqCategories}
-                  />
-                )}
-              {this.props.contentLoaded && (
-                <React.Fragment>
-                  <View spacer={true} extralarge={true} />
-                  <InstabugAssistanceComponent
-                    requestAssistance={this.props.onRequestAssistance}
-                  />
-                  <View spacer={true} />
-                  <H3>{I18n.t("instabug.contextualHelp.title2")}</H3>
-                  <View spacer={true} />
-                  <Text>
-                    {`${I18n.t("instabug.contextualHelp.descriptionLink")} `}
-                    <TouchableWithoutFeedback
-                      onPress={() => openLink(I18n.t("global.ioWebSite"))}
-                    >
-                      <Text link={true}>{I18n.t("global.ioWebSite")}</Text>
-                    </TouchableWithoutFeedback>
-                  </Text>
-                </React.Fragment>
+        {!content && (
+          <View centerJustified={true}>
+            <ActivityIndicator color={themeVariables.brandPrimaryLight} />
+          </View>
+        )}
+        {content && (
+          <Content
+            contentContainerStyle={styles.contentContainerStyle}
+            noPadded={true}
+          >
+            <H3>{customizedTitle}</H3>
+            <View spacer={true} />
+            {customizedContent}
+            <View spacer={true} />
+            {props.faqCategories &&
+              props.contentLoaded && (
+                <FAQComponent
+                  onLinkClicked={props.onLinkClicked}
+                  faqCategories={props.faqCategories}
+                />
               )}
-              {this.props.contentLoaded && <EdgeBorderComponent />}
-            </Content>
-          )}
-          <BetaBannerComponent />
-        </Container>
-      </Modal>
-    );
-  }
-}
+            {props.contentLoaded && (
```

Comment:
This logic should be updated because if there are some remote data available no assistance button will be rendered

<img src="https://user-images.githubusercontent.com/822471/84745825-6a903b00-afb5-11ea-9ac3-881a508f15b0.png" height="600"/>

Reply:
refined logic introduced within  https://github.com/pagopa/io-app/pull/1898/commits/732737daf8e67773abbeb723ee18a72f1004a450